flink-python/pyflink/table/udf.py

```diff
@@ -313,6 +435,70 @@ def _create_judtf(self):
         return j_table_function
 
 
+class UserDefinedAggregateFunctionWrapper(UserDefinedFunctionWrapper):
```

Comment:
Currently we have UserDefinedAggregateFunctionWrapper, UserDefinedTableFunctionWrapper, UserDefinedScalarFunctionWrapper, it seems to me that most of the implementation is duplicate. Could we refactor them a bit?

Reply:
Yes. Make sense